roaring/roaring.go

```diff
@@ -916,8 +924,8 @@ func (c *container) bitmapCountRange(start, end uint32) int {
 
 	// Count partial ending word.
 	if int(j) < len(c.bitmap) {
-		if off := end % 64; off != 0 {
-			n += popcount(c.bitmap[j] >> off)
+		if off := 64 - (end % 64); off != 0 {
```

Comment:
in this case, `off` will never be `0`.
this check should probably be `off != 64` (it's currently not affecting results because it's shifting out the entire 64 bits anyway and getting a count of `0`, but it's an unnecessary operation)